draft-ietf-acme-acme.md

```diff
@@ -2485,7 +2489,9 @@ domain by verifying that the resource was provisioned as expected.
 5. Verify that key authorization provided by the HTTP server matches the key
    authorization stored by the server.
 
-The server SHOULD follow redirects when dereferencing the URL.
+The server SHOULD follow redirects when dereferencing the URL.  See
+{{integrity-of-authorizations}} for security considerations related
+to redirects.
```

Comment:
Perhaps this is getting into my ballot remark instead of Alexey's, but the security considerations mostly tell you about risks of following the redirect; do we want to say anything about the advantages/which cases it is useful for?